app/views/submissions/view.html.erb

```diff
@@ -70,9 +70,18 @@
          <link rel="prerender" href= <%= url_for([:view, @course, @assessment, @prevSubmission])%> />
       <% end %>
     </div>
-    <div class="col s8 center-align valign-wrapper">
+    <div class="col s8 center-align valign-wrapper submission-controls">
       <div style="display: inline-block">
-      <%= @submission.course_user_datum.email %> (v<%= @submission.version %>) [<%= @curSubmissionIndex+1 %>/<%= @latestSubmissions.length %>]</div>
+      <%= @submission.course_user_datum.email %> [<%= @curSubmissionIndex+1 %>/<%= @latestSubmissions.length %>]</div>
+      <strong>version:</strong>
+      <select class="browser-default" onchange="location = this.value">
+        <% for submission in @userVersions do %>
+          <option <% if @submission.version == submission.version %> selected <% end %> value=<%= url_for([:view, @course, @assessment, submission]) %>>
+            <%= submission.version %>
+          </option>
+        <% end %>
+      </select>
+      <strong>/<%= @userVersions.length %></strong>
```

Comment:
suggest changing the inline-block html to (to move the version number closer to the name), so that they don't get confused by the other number representing which student this is
```
<div style="display: inline-block">
<%= @submission.course_user_datum.email %>,
<strong>version:</strong>
<select class="browser-default" onchange="location = this.value">
  <% for submission in @userVersions do %>
    <option <% if @submission.version == submission.version %> selected <% end %> value=<%= url_for([:view, @course, @assessment, submission]) %>>
      <%= submission.version %>
    </option>
  <% end %>
</select>
<strong>/ <%= @userVersions.length %></strong>
[<%= @curSubmissionIndex+1 %>/<%= @latestSubmissions.length %>]
</div>
```

<img width="683" alt="image" src="https://user-images.githubusercontent.com/5773562/165594031-50ab01ff-3962-4f93-9fef-410452349af1.png">

Reply:
Have shifted the student number to the front